libp2p/standard_setup.nim

```diff
@@ -24,11 +22,12 @@ proc newStandardSwitch*(privKey = none(PrivateKey),
                         address = MultiAddress.init("/ip4/127.0.0.1/tcp/0"),
                         triggerSelf = false,
                         gossip = false,
+                        secureManagers: openarray[string] = ["noise", "secio"],
```

Comment:
lets use a `seq` instead of strings, similar to how we do it with `transportFlags: set[ServerFlags] = {}` right below it?